src/main/scala/com/codacy/client/bitbucket/client/RequestResponse.scala

```diff
@@ -8,14 +8,21 @@ sealed trait RequestResponse[+A] {
 
   def flatMap[B](f: A => RequestResponse[B]): RequestResponse[B] = {
     this match {
-      case SuccessfulResponse(a) => f(a)
+      case SuccessfulResponse(a, _, _, _, _, _) => f(a)
       case e: FailedResponse => e
     }
   }
 
 }
 
-case class SuccessfulResponse[A](value: A) extends RequestResponse[A]
+case class SuccessfulResponse[A](
+    value: A,
+    size: Option[Int] = None,
+    pageLen: Option[Int] = None,
+    page: Option[Int] = None,
+    next: Option[String] = None,
+    previous: Option[String] = None
+) extends RequestResponse[A]
 
 case class FailedResponse(message: String) extends RequestResponse[Nothing]
 
```

Comment:
We need to add the extra fields on the   def success[A](a: A): RequestResponse[A] = SuccessfulResponse(a) or **deprecate / remove it**